pychemy/tests/test_peptide_sets.py

```diff
@@ -192,6 +194,66 @@ def test_balanced_sets(self):
     self.run_balanced_sets(max_set_size = 25)
     self.run_balanced_sets(unique = 2)
 
+###############################
+
+  def test_calc_props_with_empty_sequence(self):
+    self.assertTrue(np.array_equal(calc_props(''), np.array([])))
+
+  def test_calc_props_with_nonempty_sequence(self):
+    correct = np.array(
+    [ 1.30000000e+01,   1.42764901e+03,   7.00000000e+00,   6.00000000e+00,
+      2.00000000e+00,   4.00000000e+00,   0.00000000e+00,   4.00000000e+00,
+      5.46153846e-02,   5.00000000e-01,  -6.38461538e-01,  -5.24615385e-01,
+      8.89230769e+00,   1.60892308e+01,   1.54207692e+01,   1.00000000e+00,
+      0.00000000e+00,   1.00000000e+00,   3.00000000e+00,   0.00000000e+00,
+      0.00000000e+00,   0.00000000e+00,   1.00000000e+00,   0.00000000e+00,
+      1.00000000e+00,   1.00000000e+00,   0.00000000e+00,   3.00000000e+00,
+      0.00000000e+00,   0.00000000e+00,   1.00000000e+00,   1.00000000e+00,
+      0.00000000e+00,   0.00000000e+00,   0.00000000e+00])
+    test = calc_props('SAMPLEPEPTIDE')
+
+    for idx, item in enumerate(correct):
+      self.assertTrue(test[idx] - item < 0.001)
```

Comment:
should use https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertAlmostEqual here